File: sdks/swift/Sources/Schemas/UpdatePlanDtoBillingInterval.swift
import Foundation

public enum UpdatePlanDtoBillingInterval: String, Codable, Hashable, CaseIterable, Sendable {
    case hourly = "HOURLY"
    case daily = "DAILY"
    case weekly = "WEEKLY"
    case monthly = "MONTHLY"
    case quarterly = "QUARTERLY"
    case yearly = "YEARLY"
}

File: sdks/php/src/Plans/Types/UpdatePlanDtoBillingInterval.php
<?php

namespace NovaBilling\Plans\Types;

enum UpdatePlanDtoBillingInterval: string
{
    case Hourly = "HOURLY";
    case Daily = "DAILY";
    case Weekly = "WEEKLY";
    case Monthly = "MONTHLY";
    case Quarterly = "QUARTERLY";
    case Yearly = "YEARLY";
}
